semantic-conventions/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Semantic Conventions build tools
+
+This document provides `semantic-convention` specific information for contributions.
+
+Read OpenTelemetry project [contributing
+guide](https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md)
+for general information about the project.
+
+## Development setup
+
+### Prerequisites
+
+- Python
+- Docker (optional)
+
+### Building
+
+Any changes to the build process should be reflected in this repositiory's `.github/workflows/semconvgen.yml` workflow, in addition to this document and the [CHANGELOG](CHANGELOG.md).
+
+For local development, it may be helpful to use an isolated environment such as [venv](https://docs.python.org/3/library/venv.html), or whatever your preferred IDE may offer. Doing so will avoid polluting your system's path with dependencies specific to this project, and allow for more reproducible builds. For detailed instructions setting up python, see [venv setup](#venv-setup)
+
+Ensure dependencies are installed and on your `PATH` before building. If you're using a terminal, you can install your dependencies by running
+
+```bash
+pip install --upgrade pip
+pip install --upgrade setuptools wheel
+pip install --requirement dev-requirements.txt
+pip install --upgrade --editable .
+```
+
+### Deployment
+
+Any changes to the deployment process should be reflected in this repositiory's `.github/workflows/semconvgen.yml` workflow, in addition to this document and the [CHANGELOG](CHANGELOG.md).
+
+We use [Docker](https://docs.docker.com/) for deployments. The configuration is specified in `Dockerfile`, and you can build an image named `semconvgen` by running
+
+```bash
+# You need to package the code before building the docker image
+pip install -U pip setuptools wheel
+pip wheel --no-deps ./ --wheel-dir ./
+
+docker build -t semconvgen .
+```
+
+### venv setup
```

Comment:
We should not describe this in such detail in the project README. Link to external documentation instead, e.g. https://docs.python.org/3/library/venv.html

Reply:
I know you put a lot on effort on it so it's OK for me if we keep all of this detailed info. BUT: I think with the `Building` section you added, and the link to the official venv docs as suggested by @Oberon00 we should be good and I'd also vote to not have all these details in here.